src/component/Nav.js

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+
+const Nav = () => (
+  <header>
+    <nav className="nav_bar">
+      <div className="logo_head">
+        <img src="/images/rocket.jpeg" alt="logo" className="logo" />
+        <p>Space Travelers Hub</p>
+      </div>
+      <div className="navigation">
+        <Link className="rocket" to="/">
+          Rocket
+        </Link>
+        <Link className="mission" to="/Missions">
+          Missions
+        </Link>
+        {' '}
+        |
+        <Link className="profile" to="/MyProfile">
+          My Profile
+        </Link>
+      </div>
+    </nav>
```

Comment:
- [ ] As stated before by the previous reviewer, Kindly indicate the active page with an underlined line.